awx/playbooks/project_update.yml

```diff
@@ -186,20 +192,22 @@
         - install_roles
 
     - block:
-        - name: detect collections/requirements.yml
+        - name: detect collections/requirements.(yml/yaml)
           stat:
-            path: '{{project_path|quote}}/collections/requirements.yml'
+            path: "{{project_path|quote}}/collections/requirements{{ item.ext }}"
+          with_items: "{{ yaml_exts }}"
           register: doesCollectionRequirementsExist
 
-        - name: fetch galaxy collections from collections/requirements.yml
+        - name: fetch galaxy collections from collections/requirements.(yml/yaml)
           command: >
-            ansible-galaxy collection install -r collections/requirements.yml
+            ansible-galaxy collection install -r collections/requirements{{ item.item.ext }}
```

Comment:
I feel like it would be nicer to use `{{ item.path }}` or something like that, because you're kind of re-constructing the path when it was already given in the prior task and saved in the result.

Reply:
I agree with this idea.

Reply:
change made